code/modules/mob/living/silicon/robot/modules/module_security.dm

```diff
@@ -37,19 +34,19 @@
 	display_name = "Security"
 	crisis_locked = TRUE
 	sprites = list(
-		"Basic" = "secborg",
-		"Red Knight" = "Security",
-		"Black Knight" = "securityrobot",
 		"Bloodhound" = "bloodhound",
 		"Bloodhound - Treaded" = "secborg+tread",
-		"Tridroid" = "orb-security"
+		"Red Knight" = "Security",
+		"Tridroid" = "orb-security",
+		"Android - Guard" = "securityrobot",
+		"Android - NCO Guard" = "securityncorobot"
 	)
 	equipment = list(
 		/obj/item/device/flash,
 		/obj/item/borg/sight/hud/sec,
 		/obj/item/handcuffs/cyborg,
 		/obj/item/melee/baton/robot,
-		/obj/item/gun/energy/gun/secure/mounted,
+		/obj/item/gun/energy/gun/mounted/mk9/mounted/rubber,
```

Comment:
this is a horrible idea

Reply:
agreed this is not good

Reply:
It is an unobtainable module that cannot be played unless its a "Crisis", which I'm not really sure when that counts.